test/circuit_breaker_test.rb

```diff
@@ -63,6 +69,26 @@ def test_once_success_threshold_is_reached_only_error_threshold_will_open_the_ci
     assert_circuit_opened
   end
 
+  def test_once_success_threshold_is_reached_only_error_threshold_will_open_the_circuit_again_without_timeout
+    resource = Semian.register(
+      :three,
+      tickets: 1,
```

Comment:
I would use replace `ticket: 1` with `bulkhead: false` to make sure there is no options from "bulkheading", and only test Circuit breaker.

Reply:
Yeah good call. Fixed up.